odl/fom/fom_example.py

```diff
@@ -0,0 +1,45 @@
+"""Simple test of implemented figure-of-merits (FOMs)
+
+Numerical test of a few implemented FOMs (mean square error, mean
+absolute error, blur, and false structures) as a function of
+increasing noise level.
+
+"""
+
+import odl
+import numpy as np
+#import fom
+import matplotlib.pyplot as plt
+
+# Discrete reconstruction space: discretized functions on the rectangle
+# [-20, 20]^2 with 300 samples per dimension.
+reco_space = odl.uniform_discr(
+    min_pt=[-20, -20], max_pt=[20, 20], shape=[100, 100], dtype='float32')
+
+# Make a parallel beam geometry with flat detector
```

Comment:
Looks like copy paste. Why does this example even need tomography in it? It is never used later